casper/src/main/resources/Registry.rho

```diff
@@ -332,115 +341,388 @@ in {
+                  realAK!(arg1, arg2, arg3, arg4)
+                }
+              } |
+              contract dispatcher(@arg1, @arg2, @arg3, @arg4, @arg5) = {
+                for (realAK <<- loc) {
+                  realAK!(arg1, arg2, arg3, arg4, arg5)
+                }
+              } |
+              contract dispatcher(@arg1, @arg2, @arg3, @arg4, @arg5, @arg6) = {
+                for (realAK <<- loc) {
+                  realAK!(arg1, arg2, arg3, arg4, arg5, arg6)
+                }
+              } |
+              contract dispatcher(@arg1, @arg2, @arg3, @arg4, @arg5, @arg6, @arg7) = {
+                for (realAK <<- loc) {
+                  realAK!(arg1, arg2, arg3, arg4, arg5, arg6, arg7)
+                }
+              } |
+              contract dispatcher(@arg1, @arg2, @arg3, @arg4, @arg5, @arg6, @arg7, @arg8) = {
+                for (realAK <<- loc) {
+                  realAK!(arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8)
+                }
+              } |
+              contract dispatcher(@arg1, @arg2, @arg3, @arg4, @arg5, @arg6, @arg7, @arg8, @arg9) = {
+                for (realAK <<- loc) {
+                  realAK!(arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8, arg9)
+                }
+              } |
+              for (@bcMap <- blessedContractLocMapCh) {
+                blessedContractLocMapCh!(bcMap.set(uri, *loc))
+              } 
+            } 
+          }|
+          for (@`rho:rchain:authKey`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:authKey`, dispatcher)
+          } |
+          for (@`rho:rchain:makeMint`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:makeMint`, dispatcher)
+          } |
+          for (@`rho:rchain:multiSigRevVault`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:multiSigRevVault`, dispatcher)
+          } |
+          for (@`rho:lang:nonNegativeNumber`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:lang:nonNegativeNumber`, dispatcher)
+          } |
+          for (@`rho:rchain:pos`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:pos`, dispatcher)
+          } |
+          for (@`rho:rchain:revVault`, @lib, @dispatcher <- insertBlessed) {
+            newInsert!(lib, `rho:rchain:revVault`, dispatcher)
+          }
+        } 
+      } |
+      {
+        let @pubKey <- 
+          // fc743bd08a822d544bfbe05a5663fc325039a44c8f0c7fbea95a85517da5c36b private
+          "04e2eb6b06058d10b30856043c29076e2d2d7c374d2beedded6ecb8d1df585dfa583bd7949085ac6b0761497b0cfd056eb3d0db97efb3940b14c00fff4e53c85bf".hexToBytes(),
+        in {
+          new
+            multisig,
+            multisigLoc,       
+            multisigDispatcher
+            // Although the names above have `multisig`, they are not `multisig` currently because 
+            // we want to simplify the process and make it multisig in the future
```

Comment:
This _multisig_ here is very confusing. Firstly, it's the same name as multi-sig REV vault contract and secondly it's not working right now as multi signature process.

Please rename it to to something like `systemContractManager`.